action/src/plugins/metric/helpers.ts

```diff
@@ -0,0 +1,73 @@
+import * as core from '@actions/core';
+import fs from 'fs';
+
+import { MetricEntry } from '../../../../types/src';
+
+export const isMetricEntry = (object: any): object is MetricEntry => {
```

Comment:
Should we validate against the JSON schema for a metric entry instead of doing this manually?

Reply:
Good point. Done.